bench/Bench.hs

```diff
@@ -0,0 +1,1022 @@
+    fieldName i = padName $ Field $ Text.pack (show i)
+    padName (Field f) = Field $ Text.replicate (fieldNameLength - Text.length f) "0" <> f
+
+rowInteger :: Int -> Int -> RowData
+rowInteger nElems intValue =
+  RowData (M.fromList $ map (\i -> (fieldName i, PLiteral (LInteger (fromIntegral intValue)))) [0..nElems] )
+  where
+    fieldNameLength = 10
+    fieldName i = padName $ Field $ Text.pack (show i)
+    padName (Field f) = Field $ Text.replicate (20 - Text.length f) "0" <> f
+
+rowString :: Int -> Int -> RowData
+rowString nElems elemLength =
+  RowData (M.fromList $ map (\i -> (fieldName i, PString (Text.replicate elemLength "."))) [0..nElems] )
+  where
+    fieldNameLength = 10
+    fieldName i = padName $ Field $ Text.pack (show i)
+    padName (Field f) = Field $ Text.replicate (20 - Text.length f) "0" <> f
+
+rowDecimal :: Int -> Decimal -> RowData
+rowDecimal nElems decimalValue =
+  RowData (M.fromList $ map (\i -> (fieldName i, PDecimal decimalValue)) [0..nElems] )
+  where
+    fieldNameLength = 10
+    fieldName i = padName $ Field $ Text.pack (show i)
+    padName (Field f) = Field $ Text.replicate (20 - Text.length f) "0" <> f
+
+rowBool :: Int -> RowData
+rowBool nElems =
+  RowData (M.fromList $ map (\i -> (fieldName i, PBool False)) [0..nElems] )
+  where
+    fieldNameLength = 10
+    fieldName i = padName $ Field $ Text.pack (show i)
+    padName (Field f) = Field $ Text.replicate (20 - Text.length f) "0" <> f
+
+rowUnit :: Int -> RowData
+rowUnit nElems =
+  RowData (M.fromList $ map (\i -> (fieldName i, PUnit)) [0..nElems] )
+  where
+    fieldNameLength = 10
+    fieldName i = padName $ Field $ Text.pack (show i)
+    padName (Field f) = Field $ Text.replicate (4 - Text.length f) "0" <> f
+
+rowTime :: Int64 -> RowData
+rowTime extraMicros =
+  RowData (M.fromList [(Field "0", PTime (PactTime.fromPosixTimestampMicros extraMicros))] )
+
+-- | Make a fake keyset with nKeys keys, each key of length keyLength, with the given
+-- predicate. Each key will be the string representation of its index, padded to the
+-- desired length.
+fakeKeySet :: Int -> Int -> KSPredicate -> KeySet
+fakeKeySet nKeys keyLength pred =
+  let key i = PublicKeyText $ Text.replicate (keyLength - length (show i)) "-" <>  Text.pack (show i)
+  in KeySet (S.fromList $ map key [0..nKeys]) pred
+
+
+getModule :: String -> EvalM ReplCoreBuiltin SpanInfo (Module Name Type ReplCoreBuiltin SpanInfo)
+getModule code = do
+  let moduleSyntax = parseMod code
+  desugarOutput <- runDesugarModule @ReplCoreBuiltin @SpanInfo moduleSyntax
+  pure $ _dsOut desugarOutput
+
+
+parseMod :: String -> Lisp.Module SpanInfo
+parseMod code =
+    let parseResult = Lisp.lexer (Text.pack code) >>= Lisp.parseModule
+    in either (error . show) id parseResult
+
+exampleModule1 :: String
+exampleModule1 = "\
+ \(module m G \
+ \  (defcap G () true) \
+ \  (defun foo()       \
+ \   \"return 10\"     \
+ \   10)               \
+ \)"
+
+
+
+coinModuleCode :: String
```

Comment:
I do prefer a dedicated file, why not `coin-v5.pact`?

Reply:
I've simply removed that benchmark. We don't really need to track it, and we can bring it back if needed. Removing it allowed me to delete this awkward copy-paste of coin-v5.repl